comp/core/tagger/tagstore/tagstore.go

```diff
@@ -49,10 +49,11 @@ type TagStore struct {
 
 // NewTagStore creates new TagStore.
 func NewTagStore() *TagStore {
-	return newTagStoreWithClock(clock.New())
+	return NewTagStoreWithClock(clock.New())
 }
 
-func newTagStoreWithClock(clock clock.Clock) *TagStore {
+// NewTagStoreWithClock is exported for testing
+func NewTagStoreWithClock(clock clock.Clock) *TagStore {
```

Comment:
why do we need to make this function public? 
it seems to be used only in the tests